neo/io/nestio.py

```diff
@@ -58,34 +57,35 @@ class NestIO(BaseIO):
     write_params = None  # writing is not supported
 
     name = 'nest'
-    extensions = ['gdf', 'dat']
+    supported_target_objects = ['SpikeTrain', 'AnalogSignal']
     mode = 'file'
 
-    def __init__(self, filenames=None):
+    def __init__(self, filenames=None, target_object='SpikeTrain',
+                 additional_parameters={}):
         """
         Parameters
         ----------
             filenames: string or list of strings, default=None
-                The filename or list of filenames to load.
+                The filename or list of filename to load.
+            target_object : string or list of strings, default='SpikeTrain'
+                The type of neo object that should be read out from the input.
+                Options are: 'SpikeTrain', 'AnalogSignal'
```

Comment:
The argument `additional_parameters` is missing documentation.

Reply:
I renamed them to be kwargs, since that is the usual python way. They are passed to numpy.loadtxt, which I also now indicated in the docstring